src/cromshell/utilities/cromshellconfig.py

```diff
@@ -23,7 +23,8 @@
     "subWorkflowMetadata",
     "subWorkflowId",
 ]
-API_STRING = "/api/workflows/v1"
+CROMWELL_API_STRING = "/api/workflows/v1"
+WOMTOOL_API_STRING = "/api/womtool/v1"
```

Comment:
One of many examples where I am open to naming suggestions. For example: here the names follow the CLI naming scheme (`cromwell` and `womtool`), but I could also see naming them after the endpoints (`workflows` and `womtool`).

Reply:
I think cromwell and womtool are very understandable.  I'm definitely used to it being called cromwell, not "workflows api"